ndsl/testing/comparison.py

```diff
@@ -305,33 +323,51 @@ def one_line_report(self) -> str:
     def report(self, file_path: Optional[str] = None) -> List[str]:
         report = []
         report.append(self.one_line_report())
-        if not self.check:
```

Comment:
was dropping the self.check intentional?

Reply:
Yeah - we are reporting everything now - bad or good. The idea behind it is that when you have cascading fields, we want to know if you are _just_ passing or if you are indeed really close.

The summary will give you a one glance result at passing or failing anyway

Reply:
understood. Makes sense